products/ward round documentation/new feature description_ward round documentation.md

```diff
@@ -0,0 +1,248 @@
+# Feature Name: Ward Round Documentation
+
+## 1. Problem Statement
+
+    - Briefly describe the current workflows' main point(s) this feature aims to address.
+
+The manner in which ward rounds are documented varies considerably between hospitals and even between different wards within the same hospital. Frequently, this documentation is merely a free text entry, which is often made hours after the ward rounds have concluded. This can result in the loss of valuable information. Additionally, the required information is not always readily available, which makes it challenging to make informed decisions and may also lead to the overlooking of important findings, which could potentially endanger the patient. The procedure discussed during ward rounds necessitates the prompt distribution of tasks to the team to ensure timely patient care. The objective of the "Ward Round Documentation" system is to integrate the required functionality to provide doctors with a comprehensive overview of relevant information and to facilitate the swift documentation and task distribution processes.
+
+___
+
+## 2. Background and Objective
+
+    - Give some background information and explain how the current process is inefficient, error-prone, or lacks functionality (e.g. anecdotes from clinical practice)
+    - Outline the specific objectives and goals of the feature.
+    - What do we aim to achieve by implementing this feature?
+    - Quantify the impact of the problem if possible (e.g., increased time spent on tasks, medication errors, scientific references are definitely welcome!).
+
+**Challenges of conventional ward round documentation (paper-based or digital but limited to free-text entries):**
+- Time constraints: Clinicians only have limited time with each patient and document the ward round.
+- Incomplete or Inaccurate Information: Lack of access to up-to-date patient records or missing information cause harm to the patient assessment.
+- Coordination and logistics: Essential multidisciplinary team members might not be available at the time of ward rounds.
+- Follow-up actions: Ensuring that all agreed-upon actions are tracked and followed up can be challenging.
+- Technology limitations: Manually searching for relevant data points can be hindered by slow, unreliable, or difficult to use systems.
+- Training and standardization: Variability in the training and practices of team members can lead to inconsistencies in patient care and documentation.
+- Interruptions and distractions: In case of interruptions, it is not easily comprehensible what tasks still need to be fulfilled
+
+**Purpose of ward round documentation software:**
+- Efficient record-keeping and time savings: Manual note-taking or repetitive documentation is not required and automating repetitive tasks reduces the time needed for documentation.
```

Comment:
suggestion to add a line for our sandbox approach -> Enable clinicians to structure their own documentation as they see fit without the need for lengthy cycles with their it.